quippy/quippy/potential.py

```diff
@@ -111,6 +111,13 @@ def __init__(self, args_str="",
         self.extra_results = {'config': {},
                               'atoms': {}}
 
+    def _get_name(self):
+        return self.name_
+
+    @property
```

Comment:
Can't this just be inherited from `Calculator`?  Seems identical to https://gitlab.com/ase/ase/-/blob/master/ase/calculators/calculator.py#L538

Reply:
This is to be compatible with the version before `ase 3.23.0`. Otherwise error will raise when use `calc.name` because it dose not exist.

Reply:
Good point - I hadn't thought about backward compatibility.  I'd suggest putting in a comment that it's being left in only for that purpose, so we remember to clean it up if/when we decide ASE pre-3.23.0 isn't worth supporting.

Reply:
OK, I have added the comments